### DISCUSSION TITLE
Integration not loaded after the home assistant core update

### DISCUSSION THREAD

**Discussion by benkwyan** [music-assistant/support#1112] [Q&A] [answered]
Hi all, as title, the music assistant integration cant be loaded after 2023.3.0 update

log:

Error handling request
4:22:33 PM – (ERROR) Music Assistant (custom integration)

Setup failed for custom integration mass: Requirements for mass not found: ['music-assistant==1.8.7.4'].
4:03:31 PM – (ERROR) setup.py

**Comment by OzGav** [comment, marked as answer]
Known issue. You need to downgrade or wait for MA 2.0

**Reply by elfrinjo** [reply]
I think we'd all appreciate some insight into MA 2.0 :)
So far, I found out that it will be an add-on and that it will be released "soon"

Do you have more details to share?

**Reply by OzGav** [reply]
Not really. It has essentially the same functionality as the current MA but without the management overhead for the devs. There are some additional features planned. If you want to find out more head on over to discord and read back through the discussions.

**Reply by mfabiani53** [reply]
I am unable to find discord server for Music Assistant... can you give me some hints how to reach it?

**Reply by OzGav** [reply]
You should be able to just use the search in discord and type “music assistant” but try this 
https://discord.gg/E6dTUC87 (this will expire on 14 March)

**Reply by adothole** [reply]
Can’t find it and invite expired. Could you please share again?

**Reply by OzGav** [reply]
https://discord.gg/GHXza5Yp

**Comment by casperjjordaan** [comment]
Yes, um....... This is VERY inconvenient to me, as it is something I use every day.

NONE of my notification are now read out over my Google speakers, and I cannot stream live radio either. In fact, the Music Assistant entity does not even start. Can we PLEASE undo the changes to the latest version, or supply a walkthrough on how to downgrade please.

**Reply by OzGav** [reply]
You misunderstand. This is not an issue with MA it is HA. So just downgrade HA.

**Reply by casperjjordaan** [reply]
OOOOHHHHHHHHH

Thank you!!!!!

How? Thanks for the help buddy!!

**Reply by OzGav** [reply]
https://www.google.com/search?q=how+to+downgrade+ha

If you made a backup before updating then restore the backup
